tests/stress.rs

```diff
@@ -4686,21 +4686,26 @@ unsafe extern "C" fn cb(
     0
 }
 
+unsafe fn create_context() -> dc_context_t {
+    let mut ctx = dc_context_new(cb, std::ptr::null_mut(), std::ptr::null_mut());
+    let dir = tempdir().unwrap();
```

Comment:
this is the issue, `dir` gets dropped when the function exits and it deletes the directory then

> Once the TempDir value is dropped, the directory at the path will be deleted, along with any files and directories it contains.

from https://docs.rs/tempdir/0.3.7/tempdir/struct.TempDir.html

So you will need to return the dir, to avoid the deletion of it

Reply:
Aaa thanks!

Reply:
So the dir would have to be returned with `ctx` in a tuple?

Reply:
yeah, that is probably the easiest solution

Reply:
I'll continue on this tomorrow.